jsapp/scss/colors.scss

```diff
@@ -1,82 +1,82 @@
 // Kobo global palette of colors.
 
 /*
- * The Kobo "gray" scale
- *
- * NOTE: the list consist of colors with same hue (225) and saturation (16).
+ * The Kobo neutral scale
```

Comment:
Just wondering, why did we change the name from gray to neutral? Gray to me is more intuitive as to what the color is trying to tell me

Reply:
After Ruth and James voted they prefer "gray" too, I undid the change :)